test/fixtures/app/gulpfile.js

```diff
@@ -25,14 +26,21 @@ var options = {
   jsonKeys: ['title']
 };
 
+gulp.task('translate', ['build'], function(){
+  return gulp.src(['build'], { cwd: __dirname })
+    .pipe(statici18n({localeDirs: [localePath]}));
+});
+
 gulp.task('translate-with-deps', ['build'], function(){
   return gulp.src(['build'], { cwd: __dirname })
     .pipe(statici18n(options));
 });
 
-gulp.task('translate', ['build'], function(){
+gulp.task('translate-with-null-default-lang', ['build'], function(){
+  var opts = clone(options);
```

Comment:
One of the options is an array, `localeDirs`, and much of these tests are created via copy/paste so best not to introduce something that can mutate the setup.

All of this can be rewritten in a more modern syntax but who has the time!?